source/framework/vx_parameter.c

```diff
@@ -17,8 +17,6 @@
 
 #include <vx_internal.h>
 
-static vx_status ownDestructParameter(vx_reference ref);
```

Comment:
Is there a reason to remove this?  Please keep in place if not

Reply:
introduced again